report.c

```diff
@@ -104,8 +104,9 @@ void report_status_message(uint8_t status_code)
 void report_debug_message(const char *s)
 {
   printPgmString(PSTR("DEBUG: "));
-  while (*s)
-    printString(s++);
+  printString(s);
+
+  printPgmString(PSTR("\r\n"));
```

Comment:
No, I recently added the DEBUG functionality (last Thursday). There is no code that is merged into Grbl that calls this function - it's for debugging GRBL only.

I do not include my own CRLF when I make use of this function.